librenms/map.jinja

```diff
@@ -55,7 +56,8 @@
     'general': {
         'user': 'librenms',
         'group': 'librenms',
-        'home': '/opt/librenms',
+        'home': '/home/librenms',
```

Comment:
We should set this to '/opt/librenms' to not "pollute" the `/home/` folder.

Reply:
@kpostrup `/opt/librenms` is the Git repository. It must not be polluted. :-)
But we could change `/home/librenms` to `/opt/librenms-home`.
(I suppose there are already installations out there. We must not break them by changing the Git repo to something like `/opt/librenms/app`.)

Reply:
@alxwr I see your point. We shouldn't use the Git repo either. I think the best option is as follows:

1. Not specifying a homedir for the librenms user.
2. Setting `- createhome: False`

On Linux systems this will set the homedir to `/home/librenms` but the folder will not be created. There is no use for a homedir for this user, we create it as system user, and it is not supposed to be logging into a shell. We can of course still run `sudo -u librenms ...`

Reply:
@kpostrup I'll give your two points a try. I hope that works for every distribution, but I'm pretty sure it will. Thanks for the feedback!

By following the Git repo it's less manual work to keep code updated. But I understand the benefits of using the official releases. Maybe we can make both ways possible in a future PR.

Reply:
I give my 2 cents, if you look at how librenms documents it's own installation,
everything points to `/opt/librenms`  even the docker installation:

see https://docs.librenms.org/Installation/

Reply:
@aboe76 I'm sorry, but I don't think I understand your last sentence. Could you please elaborate a bit?

Reply:
@alxwr I meant with the last sentence: 
if you install librenms from git, and you want to update it, you only pull and checkout new version. you will never push and commit any changes back to librenms upstream repo...

Reply:
@aboe76 Thanks for the clarification. That's true. But nevertheless I ran into problems when setting `/opt/librenms` as $HOME. It's just cleaner the way @kpostrup suggested.